webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -76,7 +74,6 @@ class StepPauschbetragPersonA(StepPauschbetrag):
     
     class InputForm(SteuerlotseBaseForm):
         person_a_requests_pauschbetrag = SelectField(
-            # TODO: This is empty string mapping of key mapping - the text is within the react component
             choices=[('yes', 'yes'),('no', '')],
```

Comment:
To be honest, I still don't get it. Why is 'yes' mapped to 'yes' and 'no' to nothing? Shouldn't this be the same? And: Plaes do add a comment, jsut without "TODO". Along the lines of `# Correct text is set by the react component`

Reply:
We discussed & change the commentary :)